superset-frontend/src/components/ImportModal/ErrorAlert.tsx

```diff
@@ -31,31 +31,36 @@ export const DOCUMENTATION_LINK = supersetTextDocs
 
 export interface IProps {
   errorMessage: string;
+  dbInstall: boolean;
 }
 
-const ErrorAlert: FunctionComponent<IProps> = ({ errorMessage }) => (
+const ErrorAlert: FunctionComponent<IProps> = ({ errorMessage, dbInstall }) => (
   <Alert
     closable={false}
     css={(theme: SupersetTheme) => antdWarningAlertStyles(theme)}
     type="error"
     showIcon
     message={errorMessage}
     description={
-      <>
-        <br />
-        {t(
-          'Database driver for importing maybe not installed. Visit the Superset documentation page for installation instructions:',
-        )}
-        <a
-          href={DOCUMENTATION_LINK}
-          target="_blank"
-          rel="noopener noreferrer"
-          className="additional-fields-alert-description"
-        >
-          {t('here')}
-        </a>
-        .
-      </>
+      dbInstall ? (
```

Comment:
`showDbInstallInstructions` or something similar? The name seems a bit cryptic to me